src/benchmarks/micro/libraries/System.IO.Hashing/CrcAppendTests.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using BenchmarkDotNet.Attributes;
+using BenchmarkDotNet.Extensions;
+using MicroBenchmarks;
+
+namespace System.IO.Hashing.Tests
+{
+    [BenchmarkCategory(Categories.Libraries)]
+    public abstract class CrcAppendTests<T>
+        where T : NonCryptographicHashAlgorithm, new()
+    {
+        protected T Crc;
+        protected byte[] Buffer;
+
+        public abstract int BufferSize { get; set; }
+
+        [GlobalSetup]
+        public void Setup()
+        {
+            Buffer = ValuesGenerator.Array<byte>(BufferSize);
+            Crc = new T();
+        }
+
+        [Benchmark]
+        public void Append()
+        {
+            Crc.Append(Buffer);
+        }
+    }
```

Comment:
Would you mind adding benchmarks for other methods? Like [[GetCurrentHash()](https://learn.microsoft.com/en-us/dotnet/api/system.io.hashing.noncryptographichashalgorithm.getcurrenthash?view=dotnet-plat-ext-7.0#system-io-hashing-noncryptographichashalgorithm-getcurrenthash)](https://learn.microsoft.com/en-us/dotnet/api/system.io.hashing.noncryptographichashalgorithm.getcurrenthash?view=dotnet-plat-ext-7.0#system-io-hashing-noncryptographichashalgorithm-getcurrenthash(system-span((system-byte)))) for example?